src/vm/jitinterface.cpp

```diff
@@ -8008,56 +8009,72 @@ CorInfoInline CEEInfo::canInline (CORINFO_METHOD_HANDLE hCaller,
     }
 
 #ifdef PROFILING_SUPPORTED
-    if (CORProfilerPresent())
     {
```

Comment:
Do we still need this extra scope?

Reply:
No, if was added for the lock holder and I forgot I was the one who added it when I removed the lock